src/main/java/org/ballerinalang/command/cmd/RemoveCommand.java

```diff
@@ -126,7 +126,8 @@ private void removeAll() {
             File folder = new File(ToolUtil.getDistributionsPath());
             File[] listOfFiles;
             listOfFiles = folder.listFiles();
-            if (listOfFiles.length == 2) {
+            if (listOfFiles.length == 2 || (listOfFiles.length == 3 && folder.toPath().resolve("installer-version").
```

Comment:
@suganyasuven Shall we put a a comment here by explaining this is checking 2 files for zip scenario and 3 files for installer scneario where additional file has to be `installer-version`?

I am again wondering, shouldn't `installer-version` as to be inside installation location instead of inside distribution?